qiita_db/support_files/patches/19.sql

```diff
@@ -0,0 +1,6 @@
+-- March 19, 2015
+-- Rename columns to be more descirptive and allow easier joins
+ALTER TABLE qiita.study_status RENAME COLUMN description TO status_description;
+ALTER TABLE qiita.portal_type RENAME COLUMN description TO portal_description;
+ALTER TABLE qiita.investigation RENAME COLUMN description TO investigation_description;
+ALTER TABLE qiita.investigation RENAME COLUMN name TO investigation_name;
```

Comment:
I'm surprised that these changes don't affect more code, but I haven't had a chance to look closely